modules/SerialCommHub/tiny_modbus_rtu.cpp

```diff
@@ -174,6 +177,9 @@ static std::vector<uint16_t> decode_reply(const uint8_t* buf, int len, uint8_t e
         case 0x0B:
             EVLOG_error << "Modbus exception: Gateway target device failed to respond";
             break;
+        default: {
```

Comment:
nit: EVerest does not seem to use blocks in the case statements by default, so remove them if you do not declare a variable.

Reply:
removed